site/docs/extensions/filter-control.md

```diff
@@ -202,7 +192,43 @@ Dependence if you use the datepicker option: [bootstrap-datepicker](https://gith
 
 - **Detail:**
 
-   If the datepicker option is set use this option to configure the datepicker with the native options. Use this way: `data-filter-datepicker-options='{"autoclose":true, "clearBtn": true, "todayHighlight": true}'`.
+   If the datepicker option is set use this option to configure the datepicker with the native options. Use this way: `data-filter-datepicker-options='{"property1":value1, "property2": value2, "property3": value3}'`.
```

Comment:
Maybe we should describe this part better, i had to read it twice to understand that the that the html attributes are meant.

Reply:
How much? I mean we can add a link to the multipleSelect options

Reply:
I guess it would be enough to add `html5` and some real examples  e.g. 

If the datepicker option is set, use this option to configure the datepicker with the native html5 options. As example you could use: `data-filter-datepicker-options='{"min": "2018-01-01", "max": "2018-12-31"}'`.

Reply:
I got confused.. Now I know what you mean.

Reply:
Documentation added